util/build_vm.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# This script is intended to install Mininet and IPMininet into
+# a brand-new Ubuntu virtual machine,
+# to create a fully usable "tutorial" VM.
+set -e
+
+export LC_ALL=C
+
+MN_INSTALL_SCRIPT_REMOTE="https://raw.githubusercontent.com/mininet/mininet/master/util/vm/install-mininet-vm.sh"
+DEPS="python \
+      python-pip \
+      python3 \
+      python3-pip
+      git"
+
+# Upgrade system and install dependencies
+sudo apt update -yq && sudo apt upgrade -yq
+sudo apt install -yq ${DEPS}
+
+# Set mininet-vm in hosts since mininet install will change the hostname
+sudo sed -i -e 's/^\(127\.0\.1\.1\).*/\1\tmininet-vm/' /etc/hosts
+
+# Install mininet
+pushd $HOME
+source <(curl -sL ${MN_INSTALL_SCRIPT_REMOTE})
+sudo pip2 install mininet/
```

Comment:
I think the `install-mininet-vm.sh` script already does that. Is there any reason for installing with pip? Do you want to ensure python2 is used ?

Reply:
Pip cannot see mininet if it is not installed through it.
And since mininet is a requirement, pip refuses to install ipmininet.

I could just install ipmininet without pip but a pip installation is cleaner than an easy_install one: https://packaging.python.org/discussions/pip-vs-easy-install/

Reply:
I see, I was using setuptools for both. As long as it is consistent.